graph/api/src/main/java/org/jboss/windup/graph/model/ArchiveModel.java

```diff
@@ -19,7 +19,7 @@
 @TypeValue(ArchiveModel.TYPE)
 public interface ArchiveModel extends FileModel
 {
-    String TYPE = "ArchiveModel:";
+    String TYPE = "ArchiveModel";
     String ARCHIVE_NAME = TYPE + "archiveName";
```

Comment:
you should have here colon you deleted in TYPE

Reply:
The colon just delimits the type discriminator from edge label. Could be -, which makes less troubles with indexes IIRC.

Reply:
yep, dash or colon, if we prefer dash as it is more convenient for indexes, we should go with dashes everywhere, do you agree @OndraZizka ?

Reply:
I don't understand why `ARCHIVE_NAME` has `TYPE` prefix, but other properties (`UNZIPPED_DIRECTORY, PARENT_ARCHIVE` don't. 

This seems like another inconsistency. Should there be `TYPE` prefix or not?

Reply:
That was the random thing to satisfy uniqueness, but here that goes as we have bunch of models, types, properties and checks for uniqueness needs to be done manually until somebody experience exceptions in runtime :(

Reply:
+1, personally, I find the prefixes to be a bit distracting.